### PR TITLE
Paste link to connect

### PR DESCRIPTION
**Pull request by IanRDavies**

### CODE REVIEW

File: apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff [hunk 1]
```diff
@@ -115,6 +115,11 @@
         <target>Добавить контакт</target>
         <note>No comment provided by engineer.</note>
       </trans-unit>
+      <trans-unit id="Add contact to start a new chat" xml:space="preserve">
+        <source>Add contact to start a new chat</source>
+        <target>Добавьте контакт чтобы отправить сообщение</target>
```
Comment (jr-simplex) [inline apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff:120]: Maybe like this?
```suggestion
        <target>Добавьте контакт, чтобы начать разговор</target>
```

File: apps/ios/ru.lproj/Localizable.strings [hunk 2]
```diff
@@ -85,6 +85,9 @@
 /* No comment provided by engineer. */
 "Add contact" = "Добавить контакт";
 
+/* No comment provided by engineer. */
+"Add contact to start a new chat" = "Добавьте контакт чтобы отправить сообщение";
```
Comment (jr-simplex) [inline apps/ios/ru.lproj/Localizable.strings:89]:
```suggestion
"Add contact to start a new chat" = "Добавьте контакт, чтобы начать разговор";
```

File: apps/android/app/src/main/res/values-ru/strings.xml [hunk 3]
```diff
@@ -227,5 +232,9 @@
     <string name="a_plus_b">a + b</string>
     <string name="colored">цвет</string>
     <string name="secret">секрет</string>
+    <string name="connect_via_link">Соединиться через ссылку</string>
+    <string name="this_string_is_not_a_connection_link">Этот текст это не ссылка-приглашение!</string>
+    <string name="you_can_also_connect_by_clicking_the_link">Вы также можете соединиться открыв ссылку там, где вы её получили. Если ссылка откроется в браузере, нажмите кнопку <b>Open in mobile app</b>.</string>
+    <string name="add_contact_to_start_new_chat">Добавьте контакт чтобы отправить сообщение:</string>
```
Comment (jr-simplex) [inline apps/android/app/src/main/res/values-ru/strings.xml:238]:
```suggestion
    <string name="add_contact_to_start_new_chat">Добавьте контакт, чтобы начать разговор:</string>
```

File: apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff [hunk 4]
```diff
@@ -470,6 +485,21 @@
...
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="Paste received link" xml:space="preserve">
+        <source>Paste received link</source>
+        <target>Вставить полученную ссылку</target>
+        <note>No comment provided by engineer.</note>
+      </trans-unit>
+      <trans-unit id="Paste the link you received into the box below to connect with your contact." xml:space="preserve">
+        <source>Paste the link you received into the box below to connect with your contact.</source>
+        <target>Чтобы соединиться, вставьте ссылку, которую вы получили от вашего контакта, в это поле.</target>
```
Comment (jr-simplex) [inline apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff:500]:
```suggestion
        <target>Чтобы соединиться, вставьте в это поле ссылку, полученную от вашего контакта.</target>
```

File: apps/ios/ru.lproj/Localizable.strings [hunk 5]
```diff
@@ -313,6 +322,15 @@
...
 "Open Settings" = "Открыть Настройки";
 
+/* No comment provided by engineer. */
+"Paste" = "Вставить";
+
+/* No comment provided by engineer. */
+"Paste received link" = "Вставить полученную ссылку";
+
+/* No comment provided by engineer. */
+"Paste the link you received into the box below to connect with your contact." = "Чтобы соединиться, вставьте ссылку, которую вы получили от вашего контакта, в это поле.";
```
Comment (jr-simplex) [inline apps/ios/ru.lproj/Localizable.strings:332]:
```suggestion
"Paste the link you received into the box below to connect with your contact." = "Чтобы соединиться, вставьте в это поле ссылку, полученную от вашего контакта.";
```

File: apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff [hunk 6]
```diff
@@ -675,6 +700,11 @@ To connect, please ask your contact to create another connection link and check
         <target>Установлено соединение с сервером, через который вы получаете сообщения от этого контакта.</target>
         <note>No comment provided by engineer.</note>
       </trans-unit>
+      <trans-unit id="You can also connect by clicking the link. If it opens in the browser, click **Open in mobile app** button" xml:space="preserve">
+        <source>You can also connect by clicking the link. If it opens in the browser, click **Open in mobile app** button</source>
+        <target>Вы также можете соединиться открыв ссылку там, где вы её получили. Если ссылка откроется в браузере, нажмите кнопку **Open in mobile app**.</target>
```
Comment (jr-simplex) [inline apps/ios/SimpleX Localizations/ru.xcloc/Localized Contents/ru.xliff:705]:
```suggestion
        <target>Вы также можете соединиться, открыв ссылку там, где вы её получили. Если ссылка откроется в браузере, нажмите кнопку **Open in mobile app**.</target>
```

File: apps/ios/ru.lproj/Localizable.strings [hunk 7]
```diff
@@ -445,6 +460,9 @@
 /* No comment provided by engineer. */
 "You are connected to the server used to receive messages from this contact." = "Установлено соединение с сервером, через который вы получаете сообщения от этого контакта.";
 
+/* No comment provided by engineer. */
+"You can also connect by clicking the link. If it opens in the browser, click **Open in mobile app** button" = "Вы также можете соединиться открыв ссылку там, где вы её получили. Если ссылка откроется в браузере, нажмите кнопку **Open in mobile app**.";
```
Comment (jr-simplex) [inline apps/ios/ru.lproj/Localizable.strings:464]:
```suggestion
"You can also connect by clicking the link. If it opens in the browser, click **Open in mobile app** button" = "Вы также можете соединиться, открыв ссылку там, где вы её получили. Если ссылка откроется в браузере, нажмите кнопку **Open in mobile app**.";
```

File: apps/android/app/src/main/res/values-ru/strings.xml [hunk 8]
```diff
@@ -166,6 +169,8 @@
     <string name="your_chat_profile_will_be_sent_to_your_contact">Ваш профиль будет отправлен\nвашему контакту</string>
     <string name="if_you_cannot_meet_in_person_scan_QR_in_video_call_or_ask_for_invitation_link">Если вы не можете встретиться лично, вы можете <b>сосканировать QR код во время видеозвонка</b>, или ваш контакт может отправить вам ссылку.</string>
     <string name="share_invitation_link">Поделиться ссылкой</string>
+    <string name="paste_connection_link_below_to_connect">Чтобы соединиться, вставьте ссылку, которую вы получили от вашего контакта, в это поле.</string>
```
Comment (jr-simplex) [inline apps/android/app/src/main/res/values-ru/strings.xml:172]:
```suggestion
    <string name="paste_connection_link_below_to_connect">Чтобы соединиться, вставьте в это поле ссылку, полученную от вашего контакта..</string>
```

File: apps/android/app/src/main/res/values-ru/strings.xml [hunk 9]
```diff
@@ -227,5 +232,9 @@
     <string name="a_plus_b">a + b</string>
     <string name="colored">цвет</string>
     <string name="secret">секрет</string>
+    <string name="connect_via_link">Соединиться через ссылку</string>
+    <string name="this_string_is_not_a_connection_link">Этот текст это не ссылка-приглашение!</string>
```
Comment (jr-simplex) [inline apps/android/app/src/main/res/values-ru/strings.xml:236]:
```suggestion
    <string name="this_string_is_not_a_connection_link">Эта строка не является ссылкой-приглашением!</string>
```

File: apps/android/app/src/main/res/values-ru/strings.xml [hunk 10]
```diff
@@ -227,5 +232,9 @@
...
+    <string name="you_can_also_connect_by_clicking_the_link">Вы также можете соединиться открыв ссылку там, где вы её получили. Если ссылка откроется в браузере, нажмите кнопку <b>Open in mobile app</b>.</string>
```
Comment (jr-simplex) [inline apps/android/app/src/main/res/values-ru/strings.xml:237]:
```suggestion
    <string name="you_can_also_connect_by_clicking_the_link">Вы также можете соединиться, открыв ссылку там, где вы её получили. Если ссылка откроется в браузере, нажмите кнопку <b>Open in mobile app</b>.</string>
```

File: apps/android/app/src/main/res/values-ru/strings.xml [hunk 11]
```diff
@@ -152,6 +153,8 @@
     <string name="icon_descr_simplex_team"><xliff:g id="appName">SimpleX</xliff:g> команда</string>
     <string name="image_descr_simplex_logo"><xliff:g id="appName">SimpleX</xliff:g> логотип</string>
     <string name="icon_descr_email">Email</string>
+    <string name="icon_checkmark_action">TODO</string>
```
Comment (jr-simplex) [inline apps/android/app/src/main/res/values-ru/strings.xml:156]:
```suggestion
    <string name="icon_checkmark_action">Проверить и выполнить</string>
```

File: apps/android/app/src/main/res/values-ru/strings.xml [hunk 12]
```diff
@@ -152,6 +153,8 @@
...
+    <string name="icon_paste_from_clipboard">TODO</string>
```
Comment (jr-simplex) [inline apps/android/app/src/main/res/values-ru/strings.xml:157]:
```suggestion
    <string name="icon_paste_from_clipboard">Вставить из буфера обмена</string>
```

Comment (jr-simplex) [reply apps/android/app/src/main/res/values-ru/strings.xml:157]: I don't see where this and `icon_checkmark_action`, were they deleted from code but not from translations?

File: apps/android/app/src/main/res/values-ru/strings.xml [hunk 13]
```diff
@@ -98,13 +98,14 @@
...
     <!-- NewChatSheet -->
-    <string name="create_QR_code_or_link__bracketed__multiline">(создать QR код или ссылку)</string>
+    <string name="create_one_time_link">Создать одноразовую ссылку</string>
+    <string name="paste_received_link">Вставить полученную ссылку</string>
+    <string name="scan_QR_code">Сканировать QR код</string>
+    <string name="to_share_with_your_contact">(чтобы отправить вашему контакту)</string>
     <string name="in_person_or_in_video_call__bracketed">(при встрече или через видеозвонок)</string>
-    <string name="create_group">Создать группу</string>
-    <string name="coming_soon__bracketed">(скоро!)</string>
+    <string name="paste_received_link_from_clipboard">TODO</string>
```
Comment (jr-simplex) [inline apps/android/app/src/main/res/values-ru/strings.xml:108]:
```suggestion
    <string name="paste_received_link_from_clipboard">(вставить ссылку из буфера обмена)</string>
```

File: apps/android/app/src/main/res/values-ru/strings.xml (hunk 11 above)
Comment (epoberezkin) [reply apps/android/app/src/main/res/values-ru/strings.xml:156]: I don't think it's used, will remove it

**Pull request merged**